types/apimocker/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for apimocker 1.1
+// Project: https://www.npmjs.com/package/apimocker
+// Definitions by: Uchenna <https://github.com/uchilaka>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { RequestHandler, Application } from 'express';
+
+export interface configOptions {
+    port?: string;
+    mockDirectory?: string;
+    allowedDomains?: string[];
+    allowedHeaders?: string[];
+    logRequestHeaders?: boolean;
+    allowAvoidPreFlight?: boolean;
+    useUploadFieldname?: boolean;
+    webServices?: any;
+    quiet?: boolean;
+}
+
+export interface ApiMocker {
+    express: Application;
+    middlewares: RequestHandler[];
+    setConfigFile: (file: string) => ApiMocker;
+    loadConfigFile: () => void;
+    setRoutes: (webServices: any) => void;
+    /**
+     * Set the route for express, in case it was not set yet
+     */
+    setRoute: (options: any) => void;
+    /**
+     * @TODO inspect Express.Application.start
```

Comment:
Did you mean to leave TODO comments in the JSDoc? People will see this in their editor completions 😄

Reply:
Woops! I was leaving notes for myself. I'll circle back and flesh out the interface more, but this covers the use case in the basic examples for initializing apimocker via node. I think this should be good to ship pending any other feedback until I can add more detail or (wishful!!) re-factor for TS.